src/codecs/html/index.ts

```diff
@@ -539,7 +541,11 @@ function generateHtmlElement(
       }),
       h('meta', { 'http-equiv': 'X-UA-Compatible', content: 'ie=edge' }),
       themeCss,
-      themeJs
+      themeJs,
+      h('script', {
+        src: 'https://unpkg.com/@stencila/components@<=1',
+        type: 'text/javascript'
+      })
```

Comment:
~~This is using unversioned JS which makes sense for now but we should maybe think about pinning it later.~~ Hah, got pinned as I was writing that!

Also, I envisaged that the Web Component JS would integrate here, into Encoda, via Thema, rather than directly. Is that the long term plan?